.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   env: { es6: true },
-  ignorePatterns: ["build/*", "jest.setup.ts"],
+  ignorePatterns: ["build/*", "api/build/*", "jest.setup.ts"],
```

Comment:
Wondering whether we should just completely ignore everything in the api folder and give it its own config for ts, eslint, etc?

Reply:
I think it would be good to have one linting for the whole repo, assuming we'd use the same setting in both projects